keps/sig-cli/3515-kubectl-explain-openapiv3/README.md

```diff
@@ -618,10 +557,14 @@ enhancement:
 
 This feature only requires the target cluster has enabled The OpenAPIV3 feature.
 
-OpenAPIV3 is Beta as of Kubernetes 1.24. This feature should not be on-by-default
-until it is GA.
+OpenAPIV3 is Beta as of Kubernetes 1.24. Thus every version of Kubernetes within skew
+should be reasonably expected to have the feature on, unless it has been explicitly
+disabled. 
+
+This feature should not be on-by-default without an automatic fallback 
+until OpenAPIV3 is GA.
 
-Users of the `--output` flag who attempt to use it against a cluster for which
+Users of the `--output plaintext` flag who attempt to use it against a cluster for which
```

Comment:
This sounds like we're going to be making a distinction between "flag defaulted to 'plaintext'" (uses fallback) and "flag explicitly set to 'plaintext'" (error). Is that true?

Reply:
Thats the way I have it in my current implementation. Personally am flexible about it. What do you think?

Reply:
That sounds reasonable to me. I just wanted to make sure I understood since that isn't stated explicitly (might be worth doing so?)

Reply:
Thanks! Added explicit details about it.